components/ledger/pkg/api/apierrors/errors.go

```diff
@@ -64,7 +65,9 @@ func coreErrorToErrorCode(err error) (int, string, string) {
 	case
 		ledger.IsValidationError(err),
 		state.IsPastTransactionError(err),
-		runner.IsNoPostingsError(err):
+		runner.IsNoPostingsError(err),
+		errors.Is(err, revert.ErrAlreadyReverted),
```

Comment:
In order to keep the same syntax, should we create function ? (not a big deal for me, but just asking !)

Reply:
yes, agreed